test/other/metadce/test_metadce_minimal_pthreads.jssize

```diff
@@ -1 +1 @@
-15320
+15563
```

Comment:
Presumable these losses could be more than regained if/when we stop including the old postMessage mechanism?   

I guess we should add an option for that at some point?  Perhaps doing it automatically on browser version?

Reply:
Right now it's possible to proxy a message to a worker before it has called `Atomics.waitAsync` to start listening for notifications, so we still have to use the old code path occasionally even on browsers that support `Atomics.waitAsync`. We could fix that in principle, but it would require waiting for the new thread to execute `Atomics.waitAsync` before returning from `pthread_create`, so it would at least moving thread lifetime management off the main thread.